### PR TITLE
Replace inline puppet class with a module, fixes issue #214

### PR DESCRIPTION
**Pull request by limed**

### CODE REVIEW

**Comment by limed** [comment]
Fixes #214

**Comment by tinnightcap** [comment]
r+

File: nubis/Puppetfile [hunk 1]
```diff
@@ -54,6 +54,10 @@ mod 'nubis/nubis_storage',
     :git => 'https://github.com/Nubisproject/nubis-puppet-storage.git',
     :ref => 'v1.0.0'
 
+mod 'nubis/nubis_mig',
```
Comment (gozer) [inline nubis/Puppetfile]: This will make the puppet class named 'nubis_mig', even though the puppet code expects it to be called 'mig'

**Comment by gozer** [comment]
I think this small patch should do the trick

```diff
--- a/nubis/Puppetfile
+++ b/nubis/Puppetfile
@@ -54,7 +54,7 @@ mod 'nubis/nubis_storage',
     :git => 'https://github.com/Nubisproject/nubis-puppet-storage.git',
     :ref => 'v1.0.0'
 
-mod 'nubis/nubis_mig',
+mod 'nubis/mig',
     :git => 'https://github.com/nubisproject/nubis-puppet-mig.git',
     :ref => 'v1.0.1-1'
```

**Comment by limed** [comment]
Opened PR #278

**Pull request merged**